time stamp
string source
string destination
string message
float64 estimated_distance
